bootstrap.sh

```diff
@@ -9,10 +9,10 @@ echo Building the build tool...
 GO111MODULE=on go get ./util/src/harvey/cmd/...
 
 echo Fetching u-root and building it...
-GO111MODULE=on go get github.com/u-root/u-root
+GO111MODULE=on go get github.com/u-root/u-root@c6181ee270c59d542825e89a7c02c8c25b30373b
```

Comment:
This fails on Linux:
```
# github.com/u-root/u-root/pkg/ulog
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:92:48: undefined: unix.SYSLOG_ACTION_CONSOLE_LEVEL
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:105:25: undefined: unix.SYSLOG_ACTION_CLEAR
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:111:22: undefined: unix.SYSLOG_ACTION_READ_ALL
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:116:22: undefined: unix.SYSLOG_ACTION_READ_CLEAR
```
I had to add `golang.org/x/sys` (with the version listed in u-root `Gopkg.lock`) as an additional argument: 
```suggestion
GO111MODULE=on go get github.com/u-root/u-root@c6181ee270c59d542825e89a7c02c8c25b30373b golang.org/x/sys@05986578812163b26672dabd9b425240ae2bb0ad
```